src/tcgwars/logic/impl/gen8/ChillingReign.groovy

```diff
@@ -353,7 +353,9 @@ public enum ChillingReign implements LogicCardInfo {
             energyCost GRASS
             attackRequirement {}
             onAttack {
-              // TODO
+              if (defending.cards.filterByType(SPECIAL_ENERGY)) {
+                new Knockout(defending).run(bg)
```

Comment:
This needs to be enclosed as a targeted effect I think? Otherwise stuff like "Unaware" won't prevent it iirc.